appserver/tests/functional/embeddedtest/pom.xml

```
@@ -26,48 +26,35 @@
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
             <version>5.8.1</version>
-            <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>fish.payara.extras</groupId>
             <artifactId>payara-embedded-${payara.profile}</artifactId>
-            <version>${payara.version}</version>
+            <version>${version}</version>
```

We use payara.version everywhere else, I'd keep it the same
```suggestion
            <version>${payara.version}</version>
```